### ISSUE TITLE
not in 1.59 beta tarball?

### ISSUE THREAD

**Issue by xnox** [boostorg/compute#485]
looks like it is missing from the 1.59 beta tarball. Did this component "make" it, or not?

**Comment by xnox** [comment]
ah:"Boost.Compute is not (yet) an official Boost library" from http://boostorg.github.io/compute/

**Comment by haahh** [comment]
Boost.Compute "made" it - see https://groups.google.com/forum/#!topic/boost-compute/zuhfo2Fro6g. It's already added on develop branch of boost super-project https://github.com/boostorg/boost. I can't find which version will be the first to include Boost.Compute.

**Comment by kylelutz** [comment]
I'm actively working to get Boost.Compute merged into the Boost `master` branch. Currently, I'm waiting on a couple open pull-requests to `boostorg/boost` (see https://github.com/boostorg/boost/pull/86 and https://github.com/boostorg/boost/pull/87) which are neccessary to get Boost.Compute integrated (according to the new library checklist at https://svn.boost.org/trac/boost/wiki/ReleasePractices/ManagerCheckList).

**Comment by xnox** [comment]
Cool! So it is in progress =)

> "But look, you found the notice didn't you?"
> "Yes," said Arthur, "yes I did. It was on display in the bottom of a locked filing cabinet stuck in a disused lavatory with a sign on the door saying 'Beware of the Leopard'."

**Comment by kylelutz** [comment]
Quick update, now just waiting on getting https://github.com/boostorg/boost/pull/96 merged.

**Comment by haahh** [comment]
https://github.com/boostorg/boost/pull/96 merged

**Comment by kylelutz** [comment]
The Boost.Compute submodule has been merged into `boostorg/master`, it should appear in the next Boost release (1.61).
